src/baldr/connectivity_map.cc

```diff
@@ -63,24 +64,22 @@ json::MapPtr to_properties(uint64_t id, const std::string& color) {
   });
 }
 
-using ring_t = std::list<PointLL>;
-using polygon_t = std::list<ring_t>;
 json::MapPtr to_geometry(const polygon_t& polygon) {
   auto coords = json::array({});
-  bool outer = true;
+  // bool outer = true;
   for (const auto& ring : polygon) {
     auto ring_coords = json::array({});
     for (const auto& coord : ring) {
-      if (outer) {
-        ring_coords->emplace_back(
-            json::array({json::fp_t{coord.first, 6}, json::fp_t{coord.second, 6}}));
-      } else {
+      // if (outer) {
+      ring_coords->emplace_back(
+          json::array({json::fp_t{coord.first, 6}, json::fp_t{coord.second, 6}}));
+      /*} else {
         ring_coords->emplace_front(
             json::array({json::fp_t{coord.first, 6}, json::fp_t{coord.second, 6}}));
-      }
+      }*/
```

Comment:
adding the points of the polygons inners in reverse order isnt needed because `to_boundary` already builds the inners with the reverse winding of the outer